config/main.py

```diff
@@ -82,7 +82,7 @@ def _get_breakout_options(ctx, args, incomplete):
     else:
         breakout_file_input = readJsonFile(breakout_cfg_file)
         if interface_name in breakout_file_input[INTF_KEY]:
-            breakout_mode_list = [v["breakout_modes"] for i ,v in breakout_file_input[INTF_KEY].items() if i == interface_name][0]
+            breakout_mode_list = [v["breakout_modes"] for i, v in list(breakout_file_input[INTF_KEY].items()) if i == interface_name][0]
```

Comment:
>list [](start = 66, length = 4)

No need to convert to list

Reply:
As mentioned above, this conversion was made by the 2to3 tool, as it errs on the side of caution. See also [here](http://diveintopython3.problemsolving.io/porting-code-to-python-3-with-2to3.html#dict). We would need to assess each of these conversion individually to determine if conversion to a list is necessary.